suite-native/module-connect-device/src/components/ConnectDeviceSreenView.tsx

```diff
@@ -20,7 +20,7 @@ export const ConnectDeviceSreenView = ({ children, style }: ConnectDeviceSreenVi
 
     return (
         <Screen
-            screenHeader={<ConnectDeviceScreenHeader shouldDisplayCancelButton={false} />}
```

Comment:
should this really be removed?

Reply:
I believe so. You can cancel it and then click to `Connect & unlock` same as if you cancel PIN matrix screen.

Reply:
Tested again and you are right, it's not ideal with TT/TS3. I'll fix that.

Reply:
fixed here 158e4ab94d